test/bin/cmd.test.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var test = require('tape');
+var utils = require('./utils.js');
+var exec = require('child_process').exec;
+
+test('cli executes without error', function (t) {
+  exec('./bin/cmd.js .', function (err, stdout, stderr) {
+    t.notOk(err, 'Expect no errors thrown');
+    t.notOk(stderr, 'Expect nothing logged to stderr');
+    t.end();
+  });
+});
+
+test('cli exits with code 1 when there are errors', function (t) {
+  utils.createFile();
+
+  exec('./bin/cmd.js ./test/bin', function (err, stdout, stderr) {
+    t.equal(err.code, 1, 'Expect process to exit with code 1');
+    t.ok(stdout.replace('\n', ''), 'Expect errors to be logged to stdout');
+    t.notOk(stderr, 'Don\'t expect anything to be logged to stderr');
+
+    utils.deleteFile();
```

Comment:
If we have a file with linter errors, we need to add it to the `.eslintignore`, otherwise it will generate errors when we run `npm run lint`.

However, if we have added it to the `.eslintignore`, it won't be linted when we run the CLI and therefore no errors will be generated.

If you can think of a better way to simulate a CLI error, I'm happy to hear about it!